### PR TITLE
🐳 The drone-sonar binary is compiled using a multi-stage docker build

### PR DESCRIPTION
**Pull request by fungiboletus**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,18 @@
+FROM golang:1.13.4-alpine as build
+RUN mkdir -p /go/src/github.com/aosapps/drone-sonar-plugin
+WORKDIR /go/src/github.com/aosapps/drone-sonar-plugin 
+COPY *.go ./
```
Comment (saitho) [inline Dockerfile:4]: I think it's better to copy all files. This way it has the same behaviour as building it locally.

**Review by saitho — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (fungiboletus) [reply Dockerfile:4]: If you prefer. If it's too many useless files at some point, you can use a `.dockerignore` file.

**Comment by saitho** [comment]
The builder will be scrapped anyways after the build is done. So I don't
additional (non Go) files matter here.

Antoine Pultier <notifications@github.com> schrieb am Fr., 22. Nov. 2019,
12:00:

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+
 FROM openjdk:8-jre-alpine
 
 ARG SONAR_VERSION=3.3.0.1492
 ARG SONAR_SCANNER_CLI=sonar-scanner-cli-${SONAR_VERSION}
 ARG SONAR_SCANNER=sonar-scanner-${SONAR_VERSION}
 
 RUN apk add --no-cache --update nodejs curl
-COPY drone-sonar /bin/
+COPY --from=build /go/src/github.com/aosapps/drone-sonar-plugin/drone-sonar /bin/
```
Comment (saitho) [inline Dockerfile:15]: I think we can move this below the line starting with `ENV`.
This might speed up building and pushing the image.

**Pull request merged**